css-three-column-layout/styles.css

```diff
@@ -0,0 +1,62 @@
+
+* {
+  background-color: #1996da;
+  font-family: Lato Arial, Helvetica, sans-serif;
+}
+
+.row {
+  display: flex;
+}
+
+.container {
+  max-width: 1200px;
+  margin: 0 auto;
+  color: white;
+  padding: 20px 40px;
+}
+
+.column-third {
+  width: calc(100% / 3);
+  padding: 25px;
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
```

Comment:
`display: flex` is typically reserved for the `row` class when using layout classes. If you need to make a `column` a flex container, that likely means you need a `row` inside this `column`